### PR TITLE
Add ASHA event emitter

### PR DESCRIPTION
**Pull request by yuyangh**

### CODE REVIEW

File: bumble/profiles/asha_service.py [hunk 1]
```diff
@@ -55,13 +55,13 @@ def __init__(self, capability: int, hisyncid: List[int], device: Device, psm=0):
...
         self.capability = capability  # Device Capabilities [Left, Monaural]
         self.device = device
-        self.emitted_data_name = 'ASHA_data_' + str(self.capability)
         self.audio_out_data = b''
         self.psm = psm  # a non-zero psm is mainly for testing purpose
 
         # Handler for volume control
         def on_volume_write(_connection, value):
             logger.info(f'--- VOLUME Write:{value[0]}')
+            self.emit('volume', str(value[0]).encode())
```
Comment (barbibulle) [inline bumble/profiles/asha_service.py:64]: why not emit the volume as an integer?

File: bumble/profiles/asha_service.py [hunk 2]
```diff
@@ -76,8 +76,10 @@ def on_audio_control_point_write(_connection, value):
                     f'volume={value[3]}, '
                     f'otherstate={value[4]}'
                 )
+                self.emit('start', str(value[0]).encode())
```
Comment (barbibulle) [inline bumble/profiles/asha_service.py:79]: why not emit the 4 arguments as a tuple? (or even a `namedtuple`)?

File: bumble/profiles/asha_service.py [hunk 3]
```diff
@@ -76,8 +76,10 @@ def on_audio_control_point_write(_connection, value):
...
             elif opcode == AshaService.OPCODE_STOP:
                 logger.info('### STOP')
+                self.emit('stop', str(value[0]).encode())
```
Comment (barbibulle) [inline bumble/profiles/asha_service.py:82]: `stop` doesn't have any arguments (and value[0] is the opcode, it doesn't seem to be useful to emit the `stop` opcode as an argument to the `stop` event.

File: bumble/profiles/asha_service.py (hunk 1 above)
Comment (yuyangh) [reply bumble/profiles/asha_service.py:64]: ack

File: bumble/profiles/asha_service.py (hunk 2 above)
Comment (yuyangh) [reply bumble/profiles/asha_service.py]: ack

File: bumble/profiles/asha_service.py (hunk 3 above)
Comment (yuyangh) [reply bumble/profiles/asha_service.py]: ack

**Review by barbibulle — APPROVED** [review]

**Review by uael — APPROVED** [review]

**Pull request merged**